mollie/Shopware6Composables
# feat: add possibility to not load Mollie script globally

fixes #4 and fixes #6

# Changes
- added new option `includeScriptGlobally` which is `true` by default. If set to false, the Mollie-scripts are lazily loaded on the first initialization of the Mollie instance. This is done by exposing a loader-function from the client-side mollie plugin, which loads the mollie scripts and returns a promise. The promise resolves, when the script has been loaded.

Should be fully backwards-compatible, as the components itself are unchanged, only the underlying plugin and composable are changed.

# Test
1. Start the dev-server without changes: when opening `localhost:3000` and `localhost:3000/without` the Mollie script should be loaded like before (server-side)
2. Make changes in Nuxt-Config: change `includeScriptGlobally` to `false`
3. Reload both pages: on the index-page, the components should load just fine. On the `/without` page, the script now should only load after the Button is clicked, not before.
